tools/TimeFileMaker.py

```diff
@@ -306,20 +307,20 @@ def from_seconds(seconds, signed=False):
     Converts a number of seconds into a string time.
     '''
     sign = ('-' if seconds < 0 else '+') if signed else ''
-    seconds = abs(seconds)
+    full_centiseconds=round(abs(seconds) * 100)
+    seconds = int(full_centiseconds) // 100
+    centiseconds = full_centiseconds - (seconds * 100)
```

Comment:
Parentheses are not necessary, but I'm happy to have it either way
```suggestion
    centiseconds = full_centiseconds - seconds * 100
```